### PR TITLE
Move deprecated romans module from stdlib to Nimble package

### PR DESCRIPTION
**Pull request by lcrees**
Repo here: https://github.com/lcrees/romans

Packages PR here: https://github.com/nim-lang/packages/pull/607

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -41,3 +41,5 @@
   what to return if the environment variable does not exist.
 - Removed PDCurses wrapper from the stdlib and published it as a separate 
   Nimble package.
+- Removed deprecated romans module from the stdlib and published as separate
```
Comment (dom96) [inline changelog.md]: Would be nice if you could link to the Nimble package repo here.

**Comment by data-man** [comment]
And why this module deprecated?
This seems very useful.

**Comment by Yardanico** [comment]
@data-man probably because it's in stdlib :)

**Comment by dom96** [comment]
> And why this module deprecated?
> This seems very useful.

The romans package? I have never needed to use it.

**Comment by data-man** [comment]
@dom96 
For converting a month number, e.g.

**Comment by andreaferretti** [comment]
As useful as this may be, it certainly seems too specialized to live in the stdlib. It will work just fine as a Nimble package

**Pull request merged**